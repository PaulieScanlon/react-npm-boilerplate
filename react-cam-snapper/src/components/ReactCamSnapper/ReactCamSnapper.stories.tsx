import React from 'react'

import { ReactCamSnapper } from './ReactCamSnapper'

export default {
  title: 'ReactCamSnapper',
  parameters: {
    component: ReactCamSnapper,
    componentSubtitle: 'React Cam Snapper',
  },
}

export const usage = () => <ReactCamSnapper />
