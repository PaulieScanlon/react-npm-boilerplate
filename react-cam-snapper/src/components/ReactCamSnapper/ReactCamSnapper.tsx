import React from 'react'

export const ReactCamSnapper = () => <div>React Cam Snapper</div>
