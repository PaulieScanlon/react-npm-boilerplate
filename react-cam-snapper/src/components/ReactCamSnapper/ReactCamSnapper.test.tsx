import { render } from '@testing-library/react'

import { usage } from './ReactCamSnapper.stories'

describe('<ReactCamSnapper />', () => {
  test('renders the component', () => {
    const { asFragment } = render(usage())
    expect(asFragment()).toMatchSnapshot()
  })
})
