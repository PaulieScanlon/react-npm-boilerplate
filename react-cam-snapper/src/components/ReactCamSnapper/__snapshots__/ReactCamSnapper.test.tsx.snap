// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactCamSnapper /> renders the component 1`] = `
<DocumentFragment>
  <div>
    React Cam Snapper
  </div>
</DocumentFragment>
`;
